python/Dockerfile

```diff
@@ -3,15 +3,18 @@ LABEL maintainer="acend"
 
 EXPOSE 5000
 
-COPY app/ /app
 RUN apk --no-cache add py3-pip py3-cryptography py3-mysqlclient curl && \
     pip3 install --upgrade pip && \
-    pip3 install flask flask_sqlalchemy prometheus_flask_exporter --no-cache-dir && \
-    adduser -D web && \
+    pip3 install flask flask_sqlalchemy prometheus_flask_exporter kubernetes flake8 --no-cache-dir
+    
+COPY app/ /app
+RUN adduser -D web && \
     chown -R web:root /app && \
-    chmod -R 2775 /app
+    chmod -R 2775 /app && \
+    python3 -m compileall /app/*.py && \
+    python3 -m compileall /app/libs/*.py \
+    flake8 /app/
```

Comment:
Why change the order of instructions? This way you add an additional layer because of 2 RUN directives instead of only 1.